smart-metastore/src/main/java/org/smartdata/metastore/utils/MetaStoreUtils.java

```diff
@@ -510,6 +512,16 @@ public static MetaStore getDBAdapter(
           }
         }
 
+        try {
+          String pw = HadoopUtils
+            .getPasswordFromHadoop(SmartConfKeys.SMART_DRUID_PASSWORD, conf);
+          if (pw != null && pw != "") {
+            p.setProperty("password", pw);
+          }
+        } catch (IOException e) {
+          LOG.info("Can not get password from hadoop, use default set.");
```

Comment:
Suggestion: LOG.warn("Can not get metastore password from hadoop provision credentials, use the one configured in druid.xml");